application/src/main/kotlin/com/henriquebarucco/ssbc/sensor/get/DefaultGetSensorByIdUseCase.kt

```diff
@@ -0,0 +1,17 @@
+package com.henriquebarucco.ssbc.sensor.get
+
+import com.henriquebarucco.ssbc.sensor.SensorGateway
+import com.henriquebarucco.ssbc.sensor.SensorId
+import com.henriquebarucco.ssbc.sensor.get.dto.GetSensorByIdOutput
+import com.henriquebarucco.ssbc.shared.exceptions.ResourceNotFoundException
+
+class DefaultGetSensorByIdUseCase(
+    private val sensorGateway: SensorGateway,
+) : GetSensorByIdUseCase() {
+    override fun execute(input: String): GetSensorByIdOutput {
+        val result =
```

Comment:
que tal chamar o retorno de sensor?


```suggestion
        val sensor =
```

pois assim você consegue passar direto no 
```
        return GetSensorByIdOutput.fromSensor(sensor)
```